test/e2e/scaffold/scaffold.go

```diff
@@ -208,3 +210,7 @@ func waitExponentialBackoff(condFunc func() (bool, error)) error {
 	}
 	return wait.ExponentialBackoff(backoff, condFunc)
 }
+
+func (s *Scaffold) WaitUntilNumPodsCreatedE(selector metav1.ListOptions, desiredCount int, retries int, interval time.Duration) error {
+	return k8s.WaitUntilNumPodsCreatedE(s.t, s.kubectlOptions, selector, desiredCount, retries, interval)
```

Comment:
What's the meaning for this encapsulation?

Reply:
Is `Scaffold`  to block terratest ? The encapsulation is for use `Scaffold ` instead of `terratest`.

Reply:
`s.WaitUntilNumPodsCreatedE` does nothing but passing parameters to `k8s. WaitUntilNumPodsCreatedE`, so i think exposing `k8s. WaitUntilNumPodsCreatedE` is OK. Refactor it until we have to add more logics for it.

Reply:
I can not do it , for `s. kubectlOptions` is private.